### DISCUSSION TITLE
Support octokit for Dart

### DISCUSSION THREAD

**Discussion by alestiago** [octokit/.github#61] [Ideas]
As a developer I would like an Octokit client for Dart.

Has this been [asked](https://github.com/orgs/octokit/discussions?discussions_q=Dart+) before? If so, where can I track the progress? If not, does this need community demand to get it done? Would the team be willing to have external contributors work on the Dart client?

**Comment by gr2m** [comment]
I cannot recall anyone asking for a Dart Octokit client yet. There are several inofficial SDKs for GitHub, e.g. Googles https://github.com/google/go-github for Go. Just please don't call it Octokit. Let us know once you have a first version ready and we can help promote it
